### ISSUE TITLE
[Help] Question on different quality settings and different quality versions of albums listed in Tidal

### ISSUE THREAD

**Issue by r4nd0m-dud3** [exislow/tidal-dl-ng#260]
### I need Help.

Hello there!

I'm sorry if I'm being stupid, but I've searched back and forth and couldn't get info I'm very curious about. What exactly happens if I set download quality, let's say, high_lossless (standard flac quality) and try to download hi-res album? I have default Tidal plan. 

Should I synchronize set download quality and album quality? Tidal is kinda confusing with so many different versions of the same album. I even tried to download "low" quality album and I still got the lossless quality (checked the spectrum graph), but it seems like the file itself is different from "lossless" quality version of the same album.

### What operating system are you seeing the problem on?

Windows

### Your settings

```json
not relevant
```

**Comment by koalcosadk** [comment]
I'm not the developer, but you'll get the different files from TIDAL's servers. If you play a song on the desktop or on the app, you'll find a badge that says "High" or "MAX" depending on the song quality, but pressing on it gives you further quality options, such as the option to drop down from Hi-res audio (MAX) to CD Quality (High), or alternatively, Low, which gives you lossy AAC if you're on a metered connection, for example. tidal-dl-ng just gives you the choice to select the quality you want from what TIDAL natively offers.

The "Low" file you downloaded shouldn't be lossless, and for further reading on how to determine if something is truly lossless, you can check out this guide: https://www.reddit.com/r/skrillex/comments/3l0yxp/how_to_determine_the_true_quality_of_an_audio_file/

**Comment by r4nd0m-dud3** [comment]
> The "Low" file you downloaded shouldn't be lossless, and for further reading on how to determine if something is truly lossless, you can check out this guide

As I've already said, I've checked the spectrum and it's definitely lossless. For example, I search for "skinny ape" (a Gorillaz song), select the "low" version of it in the list while having "high_lossless" set as download quality, hit "download" and it simply downloads a lossless version. That's counter-intuitive behaviour I can't understand on my own, so I'm curious and that's why I opened this issue.

**Comment by yerbas07** [comment]
In this program, when I search for an album, I get different results for the same one, one of them may be LOW. Well, that LOW corresponds to the ATMOS version, however, when I proceed to download it, I don't get an ATMOS file, why?

**Comment by koalcosadk** [comment]
> In this program, when I search for an album, I get different results for the same one, one of them may be LOW. Well, that LOW corresponds to the ATMOS version, however, when I proceed to download it, I don't get an ATMOS file, why?

@yerbas07 Atmos isn't supported through this program. I've heard there are Telegram bots that can download Atmos tracks, or you could try finding another ripping tool.

**Comment by koalcosadk** [comment]
> > The "Low" file you downloaded shouldn't be lossless, and for further reading on how to determine if something is truly lossless, you can check out this guide
> 
> As I've already said, I've checked the spectrum and it's definitely lossless. (edit: for some reason I assumed I could see this on the spectral analysis, but forgot that "low" is a vague term and apparently it can be mp3 320, which in turn can be identical to flac on the spectrum. so here comes the second argument that the developer stated that tidal-dl-ng simply grabs the stream as is with no re-encoding - then why is the "low" file's size so big (34 mb) and bitrate shown as 1003 kbps?)
> 
> For example, I search for "skinny ape" (a Gorillaz song), select the "low" version of it in the list while having "high_lossless" set as download quality, hit "download" and it simply downloads a lossless version. That's counter-intuitive behaviour I can't understand on my own, so I'm curious and that's why I opened this issue.

@r4nd0m-dud3 I've recreated this myself but my best guess as a user is that it falls back to a lossless version on TIDAL. The same thing happened with another Low track that was available in FLAC through another version of the same release. If you download, for example, Still Life by Opeth, you get a 96kbps file from TIDAL as that's the highest quality available for it on their servers.

**Comment by yerbas07** [comment]
> > In this program, when I search for an album, I get different results for the same one, one of them may be LOW. Well, that LOW corresponds to the ATMOS version, however, when I proceed to download it, I don't get an ATMOS file, why?
> 
> @yerbas07 Atmos isn't supported through this program. I've heard there are Telegram bots that can download Atmos tracks, or you could try finding another ripping tool.

What ripping tool do you recommend? Relying on a BOT is not what I need.

**Comment by koalcosadk** [comment]
> What ripping tool do you recommend? Relying on a BOT is not what I need.

You can use TIDAL-Media-Downloader and use this comment as to how to get it to download E-AC3: https://github.com/yaronzz/Tidal-Media-Downloader/issues/882#issuecomment-1794664042.

I tried this myself just now and downloaded Atmos, so it should work for you.

**Comment by r4nd0m-dud3** [comment]
> falls back to a lossless version on TIDAL

Yeah it seems like it, but I'm still wondering which version of lossless track it falls back to, because there may be several lossless versions of the same album (explicit or not, different exclusive content, different mastering).

**Comment by snopptobbe** [comment]
In short its "recording of lossless playback" so looking at spectrum is missleading. no offense to developers, i like it anyways and yet the sound quality is all good and nothing to complain about at all.
